src/handlers/issue/generate-permits.ts

```diff
@@ -51,37 +52,35 @@ async function generateComment(totals: TotalsById, issue: GitHubIssue, config: B
 
     const beneficiaryAddress = data.length > 0 ? data[0].wallets.address : "";
 
-    const permits = [];
-    const permit = await generatePermit2Signature({
+    const erc20Permits = [];
```

Comment:
I think it makes sense to make the variable names shorthand like this `erc20Permits` instead of `erc20CompliantPermits` probably.